tst: add missing skip_internet for dns utils

wagner-intevation opened the pull request:
the dns-related tests won't work in test/build environments without internet access

kamil-certat commented on intelmq/tests/lib/test_utils.py:
```diff
@@ -367,20 +368,23 @@ def test_load_configuration_yaml_invalid(self):
         with self.assertRaises(ScannerError):
             utils.load_configuration(filename)
 
+    @skip_internet()
```
This test mocks the dns methods, it doesn't call anything.

kamil-certat commented on intelmq/tests/lib/test_utils.py:
```diff
@@ -367,20 +368,23 @@ def test_load_configuration_yaml_invalid(self):
 
         query_mocks.assert_called_once_with("example.com", "any", other="parameter")
 
+    @skip_internet()
```
This test mocks the dns, it doesn't call anything.

wagner-intevation committed:
tst: add missing skip_internet for dns utils

the dns-related tests won't work in test/build environments without
internet access
